### PR TITLE
Add terminal-notifier in order to notify when synced

### PR DESCRIPTION
**Pull request by midN**
I don't know if you want to merge it or is it any useful for anybody else.

For me personally i found the need to get notified whenever files were synced so i know i can safely refresh page during development now.
So that's why `terminal-notifier` was added.

It can be made optional if there are some objections though

### CODE REVIEW

**Comment by EugenMayer** [comment]
could you explain, what happens, when the notifier is executed? I am absolutely interested in this feature, but could we have it, for now, as an opt int, so creating a sync-option for "notify" and then only do this, when this option is enabled?

**Comment by EugenMayer** [comment]
Ups, forgot something: Thank you for the contribution!

**Comment by midN** [comment]
@EugenMayer - When the notifier is executed, it sends a simple Mac/Apple notification, which looks like this *( Right Top Corner)*:
Notifier

Agreed, sync-option sounds correct way to go, i will add it and update PR

**Comment by EugenMayer** [comment]
eventhough i like the idea, it could actually get a bit noisy, so lets make the option and go with it, great idea having those notifciations

**Comment by midN** [comment]
@EugenMayer - Added it as an config Option, was going to make it as command-line variable, but since they are not Globally defined anywhere it seemed too extreme, so went for option thingy. I added it to `example/docker-sync.yml` as well

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.0.14
+0.0.15
```
Comment (EugenMayer) [inline VERSION]: remove this please

File: example/docker-sync.yml [hunk 2]
```diff
@@ -16,6 +16,9 @@ options:
 syncs:
   # IMPORTANT: this name must be unique and should NOT match your real application container name!
   fullexample-sync:
+    # enable terminal_notifier. On every sync sends a Terminal Notification regarding files being synced. ( Mac Only ).
+    # good thing incase you are developing and want to know exactly when your changes took effect.
+    terminal_notifier: true
```
Comment (EugenMayer) [inline example/docker-sync.yml]: please use a semantic name like "notify" here, thanks

File: example/docker-sync.yml [hunk 3]
```diff
@@ -74,4 +77,4 @@ syncs:
     sync_userid: 'from_host'
     unison-dualside_host_ip: 'auto'
     unison-dualside_host_server_port: 6000
-    sync_strategy: 'unison-dualside'
\ No newline at end of file
+    sync_strategy: 'unison-dualside'
```
Comment (EugenMayer) [inline example/docker-sync.yml]: could you remove this

**Comment by EugenMayer** [comment]
that looks great already, please just adjust the changes / names and we are good too merge

File: example/docker-sync.yml (hunk 3 above)
Comment (midN) [inline example/docker-sync.yml]: It's a part of `unison-dualside-sync`, didn't change anything here, my vim removed empty-spaces/newline at the end of file, sorry for that :)

**Comment by midN** [comment]
@EugenMayer - Adjusted names and removed `VERSION` change

File: .gitignore [hunk 4]
```diff
@@ -52,3 +52,5 @@ build-iPhoneSimulator/
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
 
+# ctags
+tags
```
Comment (EugenMayer) [inline .gitignore]: why do we need this?

Comment (midN) [inline .gitignore]: It's great to have `ctags` incase somebody uses them for development, in my case i use it in combination with vim + autocomplete to jump between methods efficiently and autocomplete

Comment (EugenMayer) [inline .gitignore]: i see, why not

**Comment by EugenMayer** [comment]
great contribution @midN thank you a lot!

**Pull request merged**